webapp/app/forms/steps/lotse/merkzeichen.py

```diff
@@ -67,13 +67,18 @@ def validate_person_a_disability_degree(self, field):
             if self.person_a_has_merkzeichen_g.data or self.person_a_has_merkzeichen_ag.data:
                 if self.person_a_has_merkzeichen_g.data:
                     input_required_message = _l('form.lotse.validation-disability_degree.merkzeichen_g_selected.required')
+                    disability_min20_message = _l('form.lotse.merkzeichen_g_selected.validation-disability_degree.min20')
                 else:
                     input_required_message = _l('form.lotse.validation-disability_degree.merkzeichen_ag_selected.required')
+                    disability_min20_message = _l('form.lotse.merkzeichen_ag_selected.validation-disability_degree.min20')
                 validators.InputRequired(input_required_message)(self, field)
+
                 if field.data and field.data < 20:
-                    raise ValidationError(_l('form.lotse.validation-disability_degree.min20'))
+                    raise ValidationError(disability_min20_message)
             else:
                 validators.Optional()(self, field)
+                if field.data and field.data < 20:
+                    raise ValidationError(_l('form.lotse.validation-disability_degree.min20'))
```

Comment:
Thanks, I thought that we might need to discuss that part because I thought that you would get the NoPauschbetragsPage if you do not set any data.  I will talk to Nadine about that part

Reply:
I talked to Nadine and the decision was the following:
- If Merkzeichen aG or G and disability degree < 20: Show specific error message
- If no Merkzeichen and disability degree between 0 and 20: Show general error message
- If no Merkzeichen and disability degree 0 or not given: Show no error message -> No pauschbetrags pages

I formatted this function a little bit to make it more explicit.